{"name": "test_01", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestIOzone' object has no attribute 'fp'", "trace": "self = <Case.test_fio.TestIOzone object at 0x00000205EAD6D090>\n\n    def test_01(self):\n>       self.fp.send('ls -l')\nE       AttributeError: 'TestIOzone' object has no attribute 'fp'\n\ntest_fio.py:6: AttributeError"}, "start": 1740022199849, "stop": 1740022199849, "uuid": "fd451ea5-a4e1-4f43-852a-b0eb8fd83a9d", "historyId": "01a55d7010f48ebe0c3cf2503891f546", "testCaseId": "01a55d7010f48ebe0c3cf2503891f546", "fullName": "Case.test_fio.TestIOzone#test_01", "labels": [{"name": "parentSuite", "value": "Case"}, {"name": "suite", "value": "test_fio"}, {"name": "subSuite", "value": "TestIOzone"}, {"name": "host", "value": "Meiyi-EXT"}, {"name": "thread", "value": "23840-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Case.test_fio"}]}